content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -142,8 +142,11 @@ client libraries.
 
 ### Go client
 
-* To get the library, run the following command: `go get k8s.io/client-go/<version number>/kubernetes`, see [INSTALL.md](https://github.com/kubernetes/client-go/blob/master/INSTALL.md#for-the-casual-user) for detailed installation instructions. See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go#compatibility-matrix) to see which versions are supported.
-* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, please import API definitions from client-go rather than from the main repository, e.g., `import "k8s.io/client-go/1.4/pkg/api/v1"` is correct.
+* To get the library, run:
+  `go get k8s.io/client-go/{{< param "version" >}}/kubernetes`
```

Comment:
this is not correct. client-go version tags do not match kubernetes release versions.

See https://github.com/kubernetes/client-go/#versioning:

> client-go follows semver. We will not make backwards-incompatible changes without incrementing the major version number.
> ...
> A consequence of this is that client-go version numbers will be unrelated to Kubernetes version numbers.